Behavioral.Automation.Bindings.UI/Abstractions/IWebElement.cs

```diff
@@ -0,0 +1,11 @@
+using Behavioral.Automation.Bindings.UI.Context;
+
+namespace Behavioral.Automation.Bindings.UI.Abstractions;
+
+public interface IWebElement
+{
+    public WebContext WebContext { get; }
+    public ElementSelector ElementSelector  { get; }
+    public string? Description { get; set; }
+    public Task ShouldBecomeVisibleAsync();
```

Comment:
Don't you think, it is better to name this kind of methods in present simple ? ShouldBecome and ShouldBe, makes all this method names bloated, for no reason.
So instead of **ShouldBecomeVisibleAsync()** , **IsVisibleAsync()**. 
What do you think ?

Reply:
Renamed